testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/ConfigurationCacheGradleExecuter.groovy

```diff
@@ -31,7 +30,7 @@ class ConfigurationCacheGradleExecuter extends DaemonGradleExecuter {
         "-D${ConfigurationCacheQuietOption.PROPERTY_NAME}=true",
         "-D${ConfigurationCacheParallelOption.PROPERTY_NAME}=true",
         "-D${ConfigurationCacheMaxProblemsOption.PROPERTY_NAME}=0",
-        "-D${ConfigurationCacheStartParameter.Options.REPORT_OUTPUT_DIR}=.gradle/configuration-cache/reports"
+        "-Dorg.gradle.configuration-cache.internal.report-output-directory=.gradle/configuration-cache/reports"
```

Comment:
Since internal-integ-testing compiles to 8 and CC now compiles to 17, we can't have this hard dependency anymore. 

As a follow-up I plan to split internal-integ-testing into `internal-integ-testing`, and `internal-cross-version-testing`, both of which would depend on a new `internal-distribution-testing`. This way we could compile `internal-integ-testing` and revert this change here